helpers/core.js

```diff
@@ -76,6 +90,18 @@ var helpers = {
 
 					if (asVariable) {
 						aliases[asVariable] = item;
+						//!steal-remove-start
+						dev.warn('can-stache: Use `{{#each items item=value}}` instead of `{{#each items as item}}`.');
```

Comment:
Is it possible to use the `asVariable` in the warning to give the exact new value that should be used?

Perhaps, you could use the `options.nodeList.expression`.